mergeFallout4.sh

```diff
@@ -0,0 +1,8 @@
+export profile="Default"
+export LocalAppData="/mnt/mediaSSD/SteamLibrary/steamapps/compatdata/377160/pfx/drive_c/users/steamuser/AppData/Local/"
+export game="Fallout4"
+export _data="/mnt/mediaSSD/SteamLibrary/steamapps/common/Fallout 4/Data"
+export data="/mnt/mediaSSD/SteamLibrary/steamapps/common/Fallout 4/RData"
```

Comment:
These scripts seem to be mostly testing/convenience scripts to help your personal workflow?  Not 100% sure these should make it to the public repo.  If others were to use them, they'd need to change all these paths to their setup, and "fight" your pathing preferences, etc.

Not the biggest deal, but for now can we perhaps move these scripts to a `TestScripts` folder or somesuch?  Just to help clarify that they're not required for Mutagen.Merge's functionality

Reply:
Good point, honestly, at some point I'm going to move the exports to a .env, setup dotenv, and make it more generic, but I think for now the scripts can just stay where they are.